zenoh/src/session.rs

```diff
@@ -102,6 +106,8 @@ impl SessionState {
             publications: Vec::new(),
             subscribers: HashMap::new(),
             queryables: HashMap::new(),
+            #[cfg(feature = "unstable")]
```

Comment:
Here we are changing the layout of the session struct. Did you verify the impact on the bindings and plugins? E.g. zenoh-c and zenoh-python?

Reply:
This doesn't actually change the Session's layout, since `SessionState` is Arc'd in it. So no impact on zenoh-c and zenoh-python apart from new features to expose :)